### PR TITLE
[Backport] Version bumps  from 1.7.0 release

### PR DESCRIPTION
**Pull request by EgorPopelyaev**
This PR backports version bumps from `1.7.0` release branch and moves related prdoc files to the appropriate folder.

### CODE REVIEW

**Review by joepetrowski — APPROVED** [review]

File: cumulus/polkadot-parachain/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "polkadot-parachain-bin"
-version = "4.0.0"
+version = "1.7.0"
```
Comment (liamaharon) [inline cumulus/polkadot-parachain/Cargo.toml:3]: why are these versions regressing?

Comment (liamaharon) [reply cumulus/polkadot-parachain/Cargo.toml:3]: we've already published 4.0 for this crate https://crates.io/crates/polkadot-parachain-bin/versions

Comment (EgorPopelyaev) [reply cumulus/polkadot-parachain/Cargo.toml:3]: We used to bump it like the node release ones. But if it is not needed during the node release, probably, it should be bumped during the crates.io release

Comment (EgorPopelyaev) [reply cumulus/polkadot-parachain/Cargo.toml:3]: I had the same question about the [workspace](https://github.com/paritytech/polkadot-sdk/blob/2556e33fb4c52ea8192de68df33c151768edc2a9/polkadot/Cargo.toml#L21) version for polkadot

**Review by Morganamilo — APPROVED** [review]

File: cumulus/polkadot-parachain/Cargo.toml (hunk 1 above)
Comment (liamaharon) [reply cumulus/polkadot-parachain/Cargo.toml:3]: > We used to bump it like the node release ones. But if it is not needed during the node release, probably, it should be bumped during the crates.io release

I'm not sure what you mean, could you please elaborate?

Comment (EgorPopelyaev) [reply cumulus/polkadot-parachain/Cargo.toml:3]: I mean that we have some bumps during the node release like the `Node_Version` or `spec_version`. And there are also version changes when the crates.io release is done. Those two versions (cumulus one and the polkadot workspace) we used to bump during the node release. So should it be done that way or should I ignore those, and they will be bumped during the crates.io release which is done by Morgan.

Comment (liamaharon) [reply cumulus/polkadot-parachain/Cargo.toml:3]: what's the reason for not leaving the cargo.toml version consistent with crates.io?

Comment (ggwpez) [reply cumulus/polkadot-parachain/Cargo.toml:3]: I think this version means nothing. There is another version in the code to set the node version @skunert or someone from @paritytech/sdk-node should know, just ping them.

**Comment by ggwpez** [comment]
Where are the version bumps for all other crates?

File: cumulus/polkadot-parachain/Cargo.toml (hunk 1 above)
Comment (EgorPopelyaev) [reply cumulus/polkadot-parachain/Cargo.toml:3]: I guess there is no reason for it. It is built in the release scripts as we used to do it before. I'll revert it and adjust the scripts.

**Comment by EgorPopelyaev** [comment]
@Morganamilo the creates release is not done yet, rigth?

**Comment by ggwpez** [comment]
> @Morganamilo the creates release is not done yet, right?

Okay then lets do that in another MR.

**Review by ggwpez — APPROVED** [review]

**Pull request merged**